src/screens/monitor/components/documentTable/components/ActionButtons.tsx

```diff
@@ -1,28 +1,24 @@
-import {Box, IconButton} from "@mui/material";
-import DownloadIcon from "@mui/icons-material/Download";
-import RemoveRedEyeIcon from "@mui/icons-material/RemoveRedEye";
-import SearchIcon from "@mui/icons-material/Search";
-import React from "react";
-import styled from "styled-components";
+import { Box, IconButton } from '@mui/material';
+import RemoveRedEyeIcon from '@mui/icons-material/RemoveRedEye';
+import SearchIcon from '@mui/icons-material/Search';
+import React from 'react';
+import styled from 'styled-components';
+import DownloadButton from 'components/downloadButton';
 
 const StyledBox = styled(Box)`
   display: flex;
 `;
 
 export const ActionButtons = () => {
-
-    return (
-        <StyledBox>
-            <IconButton>
-                <DownloadIcon fontSize='small' />
-            </IconButton>
-            <IconButton>
-                <RemoveRedEyeIcon fontSize='small' />
-            </IconButton>
-            <IconButton>
-                <SearchIcon fontSize='small' />
-            </IconButton>
-        </StyledBox>
-    )
-}
-
+  return (
+    <StyledBox>
+      <DownloadButton fileUrl='https://www.w3.org/WAI/ER/tests/xhtml/testfiles/resources/pdf/dummy.pdf' />
```

Comment:
I know this is just an example pdf, but in the real life scenario, the path to the file will come from the document object and will be passed TO the ActionButtons component. Hence, I suggest using a prop for the `downloadPath` in this component and pass it from the context or at least from the  `documentTable`

Reply:
As discussed in today's weekly sync, there's no file path that we can use for now. We'll have another ticket to address this once it's being sent from the server.